holonote/annotate/display.py

```diff
@@ -423,13 +421,21 @@ def get_indices_by_position(self, **inputs) -> list[Any]:
             iter_mask = (
                 (df[f"start[{k}]"] <= v) & (v < df[f"end[{k}]"]) for k, v in inputs.items()
             )
+            subset = reduce(np.logical_and, iter_mask)
+            out = list(df[subset].index)
         elif "point" in self.region_format:
```

Comment:
This should be `elif "point-point" == self.region_format` (please double check this) and then add the previous code for other cases (`point` and `point-range`).